sale_stock_refund_picking/models/stock_move.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from openerp import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    to_refund_so = fields.Boolean(
+        "To Refund in SO",
```

Comment:
Better to call it `to_refund_sale`, as there's no need to put cryptic abbreviations.